workflow.swift

```diff
@@ -56,14 +69,15 @@ wait(ready) {
         printf("swift: The launched application did not succeed.");
     }
 
-    program2 = "stage_write/stage_write";
-    arguments2 = split("heat.bp staged.bp %s \"\" MPI \"\"" % rmethod , " ");
-    printf("size: %i", size(arguments2));
-    printf("swift: launching: %s", program2);
-    exit_code2 = @par=swproc launch(program2, arguments2);
-    printf("swift: received exit code: %d", exit_code2);
-    if (exit_code2 != 0)
+    if(rmethod == "BP")
     {
-        printf("swift: The launched application did not succeed.");
+        wait (exit_code1)
```

Comment:
I think that we need to separate out the file & staging examples.  In the current version of ADIOS (as per discussion with Norbert), there is no read/write lock around the index.  Which means that if you use files to connect heat_transfer and stage_write, there's a race condition around access to the index, and if you hit it wrong, the reader will blow up.

So the best choice IMO is to restrict to Flexpath and Dataspaces right now.  A traditional swift script, where heat_transfer is allowed to complete before executing stage_transfer would work here, too, but doesn't seem to meet with the requirements of CODAR.  Even thought the MPI file transport would most likely work, I'd rather not distribute an example that we know has a chance of blowing up even if the user does everything correctly.